constants/csv_parser.ts

```diff
@@ -2,14 +2,15 @@ import { Question, Answer } from '../models'
 import fs from 'fs'
 import path from 'path'
 import { parse } from 'csv-parse/sync'
+import { QuestionType } from '../models/question'
 
 // The type containing columns that we care about in the Flowchart CSV
 type CsvType = {
   Id: string
   Name: string
   'Line Source': string
   'Line Destination': string
-  'Text Area 1': string
+  'Text Area 1': QuestionType | 'TOOLTIP-TEXT'
```

Comment:
I don't know if this should be changed in our CsvType. Text Area 1 can contain quite literally anything (and actually also contains the question text itself). I'm not sure what it would do when parsing something like 'Yes' in as a QuestionType | 'TOOLTIP-TEXT'. Also, this doesn't account for 'TOOLTIP-HOVER-TEXT' at all.

Reply:
oooo ok i'll change it back